src/sst/core/eli/elementbuilder.h

```diff
@@ -147,12 +127,9 @@ struct InstantiateBuilder
 {
     static bool isLoaded() { return loaded; }
 
-    static const bool loaded;
+    static inline const bool loaded = Base::Ctor::template add<T>();
```

Comment:
`inline` can be used with `static` class data members in C++17, and they can be initialized in-class. This prevents multiple definitions which could occur in the old code if the variable was ODR-used (had its address taken), since it was defined in the header file.

`static constexpr` member variables are `inline` by default, but the initializer here is not `constexpr` (it possibly could be made so, if the `operator()` and its return value were `constexpr`).